core/spec/spec_helper.rb

```diff
@@ -65,6 +65,8 @@
       reset_spree_preferences
     rescue Errno::ENOTEMPTY
     end
+
+    create(:store, default: true)
```

Comment:
@szymoniwacz this will definitely break a lot of specs :)

Reply:
@damianlegawiec It did.. I need to work on that.